soccer/planning/Path.hpp

```diff
@@ -90,6 +90,17 @@ class Path {
      */
     virtual std::unique_ptr<Path> clone() const = 0;
 
+    virtual void setDebugText(QString string) {
+        _debugText = std::move(string);
+    }
+
+    virtual void drawDebugText(SystemState* state, const QColor& color = Qt::darkCyan,
```

Comment:
These probably don't need to be virtual - it seems like something that doesn't really need to be overridden by subclasses.